independent-projects/qute/core/src/main/java/io/quarkus/qute/Locates.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.qute;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Enables repeating of {@link Locate}.
+ */
+@Target(ElementType.TYPE)
+@Retention(RUNTIME)
+public @interface Locates {
```

Comment:
I think that it would make sense to place this container annotation in the `Locate` class.